### DISCUSSION TITLE
NET::ERR_CERT_AUTHORITY_INVALID error

### DISCUSSION THREAD

**Discussion by peraknezevic** [vercel/community#5623] [Help]
### Summary

I have a domain on Vercel that's been working without issues for months. 
I made no changes on Namecheap since initially adding vercel nameservers. Today I pushed some changes to a website via github and when I tried to access website I'm getting NET::ERR_CERT_AUTHORITY_INVALID error in all browsers. (not sure if these changes have anything to do with the issue I just noticed this after pushing to git today)
Website is negabiljaka.com. Any idea what's happening? 

### Steps to Reproduce

NET::ERR_CERT_AUTHORITY_INVALID
Subject: yoyoytesitgn.testinsfssss
Issuer: LucidLink BE CA
Expires on: 29 Oct 2024
Current date: 26 Jan 2024

**Comment by amyegan** [comment]
Hey @peraknezevic. I don't see any problem when I visit the site. The problem could be specific to your environment. I have seen this error in the past caused by a VPN, ISP, proxy, and other security tools.
